cmd/end2endtest/helpers.go

```diff
@@ -564,3 +573,16 @@ func votesToBigInt(votes []uint64) []*types.BigInt {
 	}
 	return vBigInt
 }
+
+func isExpectedError(err error, status string) bool {
```

Comment:
this name is misleading since it sounds like a general thing but it's actually specific to the Uninterruptible test.

so please move this to `lifecycle.go`

you could even include this inside the first `go func`, either as a var 
`isExpectedError := func(err error, status string) bool { ... }`
or not even a func, a simple struct like you did for `transitions`